changelog/4.feature.rst

```diff
@@ -0,0 +1,2 @@
+Creation of :class:`~sunkit_pyvista.plotter.SunpyPlotter` class which allows for plotting of a :class:`~sunpy.map.GenericMap` through pyvista.
+Added example for :ref:`sphx_glr_generated_gallery_plot_map_in_3d.py`, showing the demonstration of the usage of the class and it's methods.
```

Comment:
No need to delete this one, but I don't think we need to bother writing changelog entries until the first release of the package.

Reply:
I think we might as well keep track of what we are adding to the package as we go and I feel like the fragments make that easy.